lib/snowpack/protocol.ex

```diff
@@ -32,7 +32,8 @@ defmodule Snowpack.Protocol do
 
   @type opts :: Keyword.t()
   @type query :: Snowpack.Query.t()
-  @type params :: [{:odbc.odbc_data_type(), :odbc.value()}]
+  @type value :: :null | term()
+  @type params :: [{atom(), [value()]}] | [{atom(), atom(), [value()]}]
```

Comment:
Would it be worthwhile to create 2 custom types for these two different versions of params?

Reply:
I wish dialyzer would be able to use the ones from `:odbc`. I'll update to be clear where these are coming from.